modules/browser/BrowserCapabilities.js

```diff
@@ -1,7 +1,9 @@
-import BrowserDetection from './BrowserDetection';
-import capabilitiesDB from './capabilities.json';
+import { getLogger } from 'jitsi-meet-logger';
+import { BrowserDetection } from 'js-utils';
 
-// TODO: All methods should use capabilities.json when possible.
+const logger = getLogger(__filename);
+
+// TODO: Move this code to js-utils.
```

Comment:
What is "this code"? The entire file?

Reply:
Yes!

Reply:
Why not do it now?

Reply:
By moving I mean moving also starting using the capabilities.json approach. It won't be 1h work and It is pretty serious change that should be well tested.